core/src/main/scala/org/apache/spark/util/SerializableConfiguration.scala

```diff
@@ -20,7 +20,9 @@ import java.io.{ObjectInputStream, ObjectOutputStream}
 
 import org.apache.hadoop.conf.Configuration
 
+import org.apache.spark.SparkContext
```

Comment:
@dongjoon-hyun,

Except for changes in the hive-thriftserver module,  this is the only place doing `import SparkContext` now.

Reply:
Thank you, @yaooqinn .